index.md

```diff
@@ -3,21 +3,23 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is a distribution of [OpenJDK](https://openjdk.org/) developed by <a href="https://sap.com">SAP</a>. It is designed to be free, cross-platform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility). This distribution serves as the default Java Runtime for SAP's numerous applications and services.
```

Comment:
```suggestion
**SapMachine** is a distribution of the [OpenJDK](https://openjdk.org/) maintained by <a href="https://sap.com">SAP</a>. It is designed to be free, cross-platform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility). This distribution serves as the default Java Runtime for SAP's numerous applications and services.
```

Added a "the" and shouldn't we rather write "maintained" instead of "developed"? Because it is fair to say that most of the development of the OpenJDK is done outside our team...